<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCardSession</class>
 <widget class="QWidget" name="NewCardSession">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Card Session</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="mainVerticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Virtual Card Reader:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="readerComboBox"/>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="VirtualCardReaderPortLineEdit">
         <property name="toolTip">
          <string>Port Number of the virtual card reader</string>
         </property>
         <property name="text">
          <string>35963</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QCheckBox" name="HideTTLCheckBox">
         <property name="toolTip">
          <string>Wait for the JCOP port to be ready automatically</string>
         </property>
         <property name="text">
          <string>Hide TTL</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="connectPushButton">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QCheckBox" name="autoConnectCheckBox">
         <property name="toolTip">
          <string>Wait for the JCOP port to be ready automatically</string>
         </property>
         <property name="text">
          <string>Auto</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>JCopTool Port:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="JCopPortLineEdit">
         <property name="toolTip">
          <string>JCOP port number</string>
         </property>
         <property name="text">
          <string>8050</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="clearPushButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QCheckBox" name="msCheckBox">
         <property name="text">
          <string>Disable MS Card</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="OutputTextEdit"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
